### PR TITLE
Us1 header

### PR DESCRIPTION
**Pull request by FLorianBlm**
Html header and footer on base.html.twig

### CODE REVIEW

**Review by Quentin-BLANC — DISMISSED** [review]

**Review by thocle86 — DISMISSED** [review]
ok

File: templates/base.html.twig [hunk 1]
```diff
@@ -3,17 +3,60 @@
...
-{% block javascripts %}
-    {{ encore_entry_script_tags('app') }}
-{% endblock %}
+    <section class="navBar">
+        <div class="logoWewelcome">
+            <a href="">
+                <img src="build\images\WeWELCOME-Logo-RVB-500px.8d7a2900.png" alt="wewelcomeLogo" width="350" height="200">
+            </a>
+        </div>
+        <div class="navButton">
```
Comment (RomainClair) [inline templates/base.html.twig]: Should probably be a nav

File: templates/base.html.twig [hunk 2]
```diff
@@ -3,17 +3,60 @@
...
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/alexandre-delalandes/?originalSubdomain=fr">
+                        <img src="build\images\linkedin_icon.d5fe90a3.png" alt="LinkedIn_Logo" width="42" height="42">
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <div class="othersLinksFooter">
```
Comment (RomainClair) [inline templates/base.html.twig]: Should probably be a nav

File: templates/base.html.twig [hunk 3]
```diff
@@ -3,17 +3,60 @@
...
+                <li><a href="">Logement</a></li>
+                <li><a href="">Contact</a></li>
+            </ul>
+            <a href="">English</a>
+        </div>
+    </section>
+    {% block body %}
+    {% endblock %}
+    <section class="footer">
+        <div class="networkFooter">
```
Comment (RomainClair) [inline templates/base.html.twig]: Should also be a nav

**Review by RomainClair — CHANGES_REQUESTED** [review]
Composer.lock shouldn't change for no valid reason

**Review by thocle86 — APPROVED** [review]
ok

**Review by CeliaBoiss — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**